lib/Echidna/Solidity.hs

```diff
@@ -7,7 +7,7 @@
 
 module Echidna.Solidity where
 
-import Control.Lens
+import Control.Lens hiding        (Indexed)
```

Comment:
why hide this?

Reply:
there's an `Indexed` from HEVM that needs using somewhere.

Reply:
but that was from when part of the code lived in Solidity.hs, i can get rid of this now